powerfit/powerfit.py

```diff
@@ -127,7 +127,11 @@ def main():
         devs = p.get_devices()
         ctx = cl.Context(devices=devs)
```

Comment:
I don't get why I put this here, this should actually be removed. Maybe that already fixes the issues you are having. If not, then the change is fine.